routes/api.php

```diff
@@ -14,6 +17,26 @@
 |
 */
 
+
 Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+//Routegroup
+Route::group(['prefix'=> 'posts', 'as' => 'post.'], function(){
+
+    Route::post('store', [PostController::class, 'store']) ->name('store');
+
+    Route::get('index', [PostController::class, 'index']) ->name('index');
```

Comment:
No need to put words index, show, update, delete.
Please follow URI like https://laravel.com/docs/10.x/controllers#actions-handled-by-resource-controller
As you can see there is no words index, show, update, delete in the URI. Except for edit.
Please apply to others below.